ui/pages/permissions-connect/snaps/snap-install/snap-install.js

```diff
@@ -140,40 +140,38 @@ export default function SnapInstall({
               permissions={requestState.permissions || {}}
               targetSubjectMetadata={targetSubjectMetadata}
             />
+            {isScrollable && !isScrolledToBottom ? (
+              <AvatarIcon
+                className="snap-install__scroll-button"
+                data-testid="snap-install-scroll"
+                iconName={IconName.Arrow2Down}
+                backgroundColor={BackgroundColor.infoDefault}
+                color={BackgroundColor.backgroundDefault}
+                onClick={scrollToBottom}
+                style={{ cursor: 'pointer' }}
+              />
+            ) : null}
           </>
         )}
-        {isEmpty && (
-          <Box
-            flexDirection={FLEX_DIRECTION.COLUMN}
-            height={BLOCK_SIZES.FULL}
-            alignItems={AlignItems.center}
-            justifyContent={JustifyContent.center}
-          >
-            <Text textAlign={TEXT_ALIGN.CENTER}>
-              {t('snapInstallRequest', [
-                <b key="1">{originMetadata?.hostname}</b>,
-                <b key="2">{snapName}</b>,
-              ])}
-            </Text>
-          </Box>
-        )}
       </Box>
       <Box
         className="snap-install__footer"
         alignItems={AlignItems.center}
         flexDirection={FLEX_DIRECTION.COLUMN}
+        style={{
```

Comment:
Is it possible to keep this in `scss` file, given that this component is already in `/snaps/` folder and does not require `only include in...`? It's probably the best practice to keep styling code like this in SCSS files if/when possible.